bumble/profiles/csis.py

```diff
@@ -0,0 +1,127 @@
+# Copyright 2021-2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# -----------------------------------------------------------------------------
+# Imports
+# -----------------------------------------------------------------------------
+from __future__ import annotations
+import enum
+import struct
+from typing import Optional
+
+from bumble import gatt
+from bumble import gatt_client
+
+# -----------------------------------------------------------------------------
+# Constants
+# -----------------------------------------------------------------------------
+
+
+class MemberLock(enum.IntEnum):
+    UNLOCKED = 0x01
+    LOCKED = 0x02
+
+
+# -----------------------------------------------------------------------------
+# Server
+# -----------------------------------------------------------------------------
+class Service(gatt.TemplateService):
+    UUID = gatt.GATT_COORDINATED_SET_IDENTIFICATION_SERVICE
+
+    def __init__(
+        self,
+        sirk: bytes,
+        set_size: Optional[int] = None,
```

Comment:
I'd recommend keeping the characteristic names exactly as in the spec, even if that's sometimes a bit long, it makes it very clear and unambiguous how the profile implementation matches the spec.
Here, we'd have  `set_identity_resolving_key`, `coordinated_set_size`, `set_member_lock` and `set_member_rank`, as listed in P.5 of the CISC profile spec.

Reply:
Done.